src/pages/DailyStatus.js

```diff
@@ -0,0 +1,86 @@
+import React, { useState, useEffect, useCallback, useMemo } from "react";
```

Comment:
please recheck is useMemo is used anywhere
```suggestion
import React, { useState, useEffect, useCallback } from "react";
```

Reply:
resolved